src/Util/Process.php

```diff
@@ -36,9 +39,31 @@ public static function run(
         string|null $workingDirectory = null,
         int|null $timeout = 5,
         callable|null $outputCallback = null,
+        bool $cleanWarningsAndDeprecations = false,
     ): string {
-        return trim((new SymfonyProcess($command, $workingDirectory, timeout: $timeout))
+        $output = trim((new SymfonyProcess($command, $workingDirectory, timeout: $timeout))
             ->mustRun($outputCallback)
             ->getOutput());
+
+        if ($cleanWarningsAndDeprecations) {
+            return self::cleanLanguageWarningAndDeprecations($output);
+        }
+
+        return $output;
+    }
+
+    private static function cleanLanguageWarningAndDeprecations(string $testOutput): string
```

Comment:
Hmm; I don't think this should be part of `Process`, it should be in `PhpBinaryPath`, since it is specifically related to PHP; the `Process` we are using for other non-PHP executables too, so it doesn't make sense in those cases IMO. Could this be moved to `PhpBinaryPath` please? :)

Reply:
Yes I wasn't sure where to put this, but I guess calling the cleaning method in `PhpBinaryPath` is better. That's updated 👍